client/my-sites/plugins/marketplace/marketplace-domain-upsell/index.tsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import DomainPicker from '@automattic/domain-picker';
+import { useShoppingCart } from '@automattic/shopping-cart';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import { domainRegistration } from 'calypso/lib/cart-values/cart-items';
+import { DomainSuggestions } from '@automattic/data-stores';
+import {
+	MARKETPLACE_FLOW_ID,
+	ANALYTICS_UI_LOCATON_MARKETPLACE_DOMAIN_SELECTION,
+} from 'calypso/my-sites/plugins/marketplace/constants';
+import { Button } from '@wordpress/components';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+import './styles.scss';
+import { getProductsList, isProductsListFetching } from 'calypso/state/products-list/selectors';
+
+type DomainSuggestion = DomainSuggestions.DomainSuggestion;
+import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+
+function Header() {
+	return (
+		<div className="marketplace-domain-upsell__header domains__header">
+			<h1 className="marketplace-domain-upsell__header marketplace-title">Choose a domain</h1>
```

Comment:
I'm assuming we're intentionally not translating those strings yet?

Reply:
Fixed by 8f9a969c32f2f1aa09768953525344b017a19cc0

Reply:
Thanks @jdc91! FWIW there are more strings we're still not translating here. Is that intentional?

Also, it's common practice to intentionally not localize strings that are not ready for production yet, because we might still iterate on them and that saves us money from translating strings that aren't guaranteed to be the final ones. That's the main reason why I was asking 😉